src/components/gameshow/RightBar.jsx

```diff
@@ -0,0 +1,248 @@
+                  {pub.name}
+                </p>
+              ))
+            ) : (
+              <p className="text-light/80">Aucun éditeur disponible</p>
+            )
+          ) : (
+            <p className="text-light/80">Loading...</p>
+          )}
+          <h3 className="text-m mt-3 font-bold">PEGI Rating</h3>
+          <div>
+            {isLoaded ? (
+              game.esrb_rating != null ? (
+                <img
+                  className="mt-1 h-16"
+                  src={ageRating(game.esrb_rating.name)}
+                  alt={game.esrb_rating.name}
+                />
+              ) : (
+                <p className="text-light/80">N/A</p>
+              )
+            ) : (
+              <p className="text-light/80">Loading...</p>
+            )}
+          </div>
+          <img
+            className="mb-5 mt-6"
+            src="/src/assets/separator.png"
+            alt="separator"
+          />
+          <h3 className="text-m mt-3 font-bold">Web site</h3>
+          {isLoaded ? (
+            game.website && game.website.length > 0 ? (
+              <a href={game.website} target="_blank" rel="noreferrer">
+                <p className="overflow-hidden text-ellipsis text-primary">
+                  {game.website}
+                </p>
+              </a>
+            ) : (
+              <p className="text-light/80">N/A</p>
+            )
+          ) : (
+            <p className="text-light/80">Loading...</p>
+          )}
+        </div>
+        <div className="rightbar w-25 container mt-5 rounded-lg border border-primary bg-primary/10 p-4">
+          <h3 className="text-m mt-3 font-bold">Buy {game.name}</h3>
+          {isLoaded ? (
+            game.stores && game.stores.length > 0 ? (
+              game.stores.map((st) => (
+                <div key={st.id}>
+                  <button className="storeButton my-2 inline-flex w-full items-center justify-center rounded bg-[#0D4F61] text-center font-bold text-light transition duration-150 hover:-translate-y-0.5 hover:bg-primary">
+                    <div className="flex w-full flex-row justify-center px-4 py-3 hover:invert">
+                      {st.store.name}
+                      <img
+                        width="25"
+                        className="mx-1.5 text-center"
+                        src={storeLogo(st.store.name)}
+                        alt={st.store.name}
+                      />
+                    </div>
+                  </button>
+                </div>
+              ))
+            ) : (
+              <p className="text-light/80">No store available</p>
+            )
+          ) : (
+            <p className="text-light/80">Loading...</p>
+          )}
+          <img
+            className="my-5"
+            src="/src/assets/separator.png"
+            alt="separator"
+          />
+          <h3 className="text-m mt-3 font-bold">
+            Editions and DLC&apos;s {game.name}
```

Comment:
Quand il y a des caractères spéciaux, il faut encapsuler dans des `{""}`
```suggestion
            {`Editions and DLC's ${game.name}`}
```